src/Azure.Functions.Cli/Actions/DurableActions/DurableCreateHelpersAction.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Colors.Net;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Helpers;
+using Azure.Functions.Cli.Interfaces;
+using static Azure.Functions.Cli.Common.OutputTheme;
+
+using Buildalyzer;
+using Buildalyzer.Workspaces;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+
+using Microsoft.Azure.WebJobs;
+using Azure.Functions.Cli.Actions.LocalActions;
+
+namespace Azure.Functions.Cli.Actions.DurableActions
+{
+    [Action(Name = "create-helpers", Context = Context.Durable, HelpText = "Create or update generated durable functions helpers")]
+    internal partial class DurableCreateHelpersAction : BaseAction
+    {
+        public const string GeneratedFileName = "DurableFunctionsHelpers.generated.cs";
+        private readonly ISecretsManager _secretsManager;
+
+        public DurableCreateHelpersAction(ISecretsManager secretsManager)
+        {
+            _secretsManager = secretsManager;
+        }
+        public override async Task RunAsync()
+        {
+            var workerRuntime = WorkerRuntimeLanguageHelper.GetCurrentWorkerRuntimeLanguage(_secretsManager);
+
+            ColoredConsole.WriteLine(AdditionalInfoColor($"Runtime: {workerRuntime}"));
+
+            if (workerRuntime != WorkerRuntime.dotnet)
+            {
+                ColoredConsole.WriteLine(ErrorColor($"create-durable-helpers does not support runtime: {workerRuntime}"));
+                return;
```

Comment:
throw here so that the exit code is set correctly, or set the exit code yourself.

Reply:
Updated this to set the exit code